src/facades/identity.ts

```diff
@@ -0,0 +1,14 @@
+import { ParticipantController } from "../controllers/identity-controllers/participant-controller";
+import { ParticipantsController } from "../controllers/identity-controllers/participants-controller";
+import { EdcController } from "../edc-controller";
+
+export class IdentityController extends EdcController {
+  get participants() {
+    return new ParticipantsController(this.inner, this.context);
+  }
+
+  get participant() {
+    return (participantId: string) =>
+      new ParticipantController(this.inner, participantId, this.context);
+  }
```

Comment:
```suggestion
  get participant(participant: string) {
    return new ParticipantController(this.inner, participantId, this.context);
  }
```

from the caller perspective the result is the same, but having a method that retuns an actual controller (as `participants` does) looks more reasonable and lean than to have the additional lambda.

Reply:
A get accessor can not have parameters ( at least in typescript to my knowledge)
I wanted to keep the getter for the facade aesthetic but we can change it to something like this instead 
```ts
  participant(participantId: string) {
    return new ParticipantController(this.inner, participantId, this.context);
  }
```

Reply:
yes, using the lamba looks like unnecessary complexity